scripts/collect_steps.py

```diff
@@ -171,6 +169,40 @@ def extract_intermediate_steps(steps):
                     print(" ".join(args), file=sys.stderr)
                     steps.update(collect_steps(args))
 
+OPRF_BREAKDOWN_KEYS = [256]
+OPRF_USER_CAP = [16, 64, 128]
+OPRF_SECURITY_MODEL = ["semi-honest"]
+OPRF_TRIGGER_VALUE = [7]
+
+def oprf_steps(steps):
+    for c in OPRF_USER_CAP:
+        for w in ATTRIBUTION_WINDOW:
+            for b in OPRF_BREAKDOWN_KEYS:
+                for m in OPRF_SECURITY_MODEL:
+                    for tv in OPRF_TRIGGER_VALUE:
+                        args = ARGS + [
+                            "-n",
+                            str(QUERY_SIZE),
+                            "-c",
+                            str(c),
+                            "-w",
+                            str(w),
+                            "-b",
+                            str(b),
+                            "-m",
+                            m,
+                            "-t",
+                            str(tv),
+                            "-o"
+                        ]
+                    print(" ".join(args), file=sys.stderr)
+                    steps.update(collect_steps(args))
+
+if __name__ == "__main__":
+    steps = set()
+    ipa_steps(steps)
+    oprf_steps(steps)
```

Comment:
Ideally, each function should return a new set and update the `steps`, rather than mutating the argument.